lib/src/main/java/io/ably/lib/http/HttpUtils.java

```diff
@@ -15,6 +15,9 @@
  */
 public class HttpUtils {
 	/* Headers */
+	public static final String X_ABLY_VERSION_HEADER = "X-Ably-Version";
+	public static final String X_ABLY_VERSION_VALUE = BuildConfig.VERSION.substring(0,
+          BuildConfig.VERSION.indexOf('.', BuildConfig.VERSION.indexOf('.') + 1));
```

Comment:
I dislike this.  If at any point you wanted to have a client library version diverge from the Ably protocol version this will no longer work.  Adding a constant is straightforward and gives us flexibility. See https://github.com/ably/ably-ruby/blob/master/lib/ably/version.rb for a similar example.